pkg/kubectl/cmd/logs.go

```diff
@@ -293,13 +289,32 @@ func (o LogsOptions) RunLogs() error {
 		return err
 	}
 
+	doneChan := make(chan bool)
+	piper, pipew := io.Pipe()
 	for _, request := range requests {
-		if err := o.ConsumeRequestFn(request, o.Out); err != nil {
-			return err
-		}
+		go func(request *rest.Request) {
```

Comment:
is there a maximum number of parallel requests we should allow?

Reply:
We need to have a default limit, with an option to change it. Otherwise by mistake we can dos the api server and that's something we don't want to happen.